### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by PowerBearMeng** [paya54/Anomaly_Detect_LSTM_VAE#4]
"Hello, when reproducing my own dataset, I often encounter situations where the training loss fluctuates drastically, and after one or two epochs, the loss becomes NaN. I wonder if you could help me with this issue. I would be very grateful!"
